util/environment.ts

```diff
@@ -0,0 +1,54 @@
+const DEFAULT_ENVIRONMENTS_REGEX: Record<EnvironmentName, string> = {
+  local: '^(:?localhost:\\d{2,5}|(?:127|192)(?:\\.[0-9]{1,3}){3})',
+  pr: '^cowfi-git-[\\w\\d-]+cowswap\\.vercel\\.app',
+  development: '^(develop.cow.fi|swap-develop.vercel.app)',
+  production: '^cow.fi$',
```

Comment:
If we want to be complete, there are more prod domains to add 
![image](https://github.com/cowprotocol/cow-fi/assets/43217/a7c16588-11e7-415c-af33-fef2332ed7d4)

Although it's probably overkill